netunnel/server/server.py

```diff
@@ -147,6 +148,24 @@ async def _setup_peers(self):
             peer = await self._load_peer_from_config(peer_config, verify_connectivity=False)
             self._peers[peer.id] = peer
 
+    async def _shutdown_peers(self):
```

Comment:
don't know if shutdown is the best terminology to use here. it's more like "detach peers", no?